int32   linear_ticks_sec
int32   angular_ticks_sec
